lib/index.js

```diff
@@ -193,7 +213,7 @@ Appboy.prototype.identify = function(identify) {
   var phone = identify.phone();
   var traits = clone(identify.traits());
 
-  window.appboy.changeUser(userId);
+  window.appboy.changeUser(userId, window.__hack_segment_appboy_change_user_sb);
```

Comment:
`changeUser` only calls `messagingReadyCallback` if it's a function, so we can just pass this variable in.